### PR TITLE
Checkerboard removal

### PR DESCRIPTION
**Pull request by holongate**
hi!
This is a quick (and easy) fix for removing your checkerboard artifact.
- Corrected kernel_size = 3 to krnel_size = 1 in second block of up_layer
- Added a final conv2D + sigmoid of (1x1x3) to restore initial image shape (this was the missing bit that created the checkerboard artifact)

(Also, upgraded to Python 3.6 and tensorflow 1.5.)

Keep the good work going on!
Have fun.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 This is a tensorflow implemenation of [Deep Image Prior](https://dmitryulyanov.github.io/deep_image_prior).
 
 # Setup
-- Install python 3.6, [tensorflow](https://www.tensorflow.org/install/) (tested on 1.5.0+), numpy, and scipy.
+- Install python ~~2.6~~ 3.6, [tensorflow](https://www.tensorflow.org/install/) (tested on ~~1.4.0~~ 1.5.0+), numpy, and scipy.
```
Comment (beala) [inline README.md]: No need for strikethroughs here (and below).

File: deepimg.py [hunk 2]
```diff
@@ -169,7 +168,19 @@ def gblur(layer):
...
+    num_outputs=3,
+    kernel_size=1,
+    stride=1,
+    padding='SAME',
+    activation_fn=tf.nn.sigmoid)
+print("Output shape: " + str(out.get_shape()))
+
+# Use built-in rms implementation
+E = tf.losses.mean_squared_error(image, gblur(out))
+#E = tf.reduce_mean(tf.squared_difference(gblur(out), image))
```
Comment (beala) [inline deepimg.py]: Can delete this rather than commenting out.

**Review by beala — CHANGES_REQUESTED** [review]
Nice work! Thanks for making this PR.

I left a couple comments. Also, feel free to make a contributors file and add your name.

**Comment by holongate** [comment]
Done!
